src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryMessages.java

```diff
@@ -6,6 +6,9 @@ private MercuryMessages() {}
   public static final String VISIBLE_MSG = "element is visible";
   public static final String INVISIBLE_MSG = "element is not visible";
 
+  public static final String COLLAPSED_MSG = "element is not collapsed";
+  public static final String EXPANDED_MSG = "element is not expanded";
+
   public static final String LIST_ITEMS_ARE_UNIQUE_MSG = "all list elements are unique";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)